README.md

```diff
@@ -12,11 +12,13 @@ To build just the vocoder itself you will need:
     - Debian
     - ALSA
 - Hardware
-    - Raspberry Pi 3 (or newer)
+    - Raspberry Pi 3 (or newer) running a 64bit OS
     - Codec Zero
 
 Additionally, to run the unit tests and static analysis suite, LLVM 15 + Tools is required. To install this on debian the automatic script from [their website](https://apt.llvm.org/) can be used.
 
+To build the doxygen documentation, the `doxygen` and `graphviz` packages are required.
```

Comment:
I think it looks messy to have these in a paragraph on their own. I think they should be added to the block of commands below. We could also make an additional list of optional software requirements in the list above.

Reply:
It matches the paragraph above.

Reply:
Added a bit to the docs section that explains installing them.